lib/hyacinth/digital_object/facets.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+module Hyacinth
+  module DigitalObject
+    module Facets
```

Comment:
I didn't see any tests for the methods in this class, but that would be good to add.